src/plugins/procdump/private.h

```diff
@@ -146,30 +150,65 @@ enum
     MMPTE_PROTOTYPE,
 };
 
+#define MM_GUARD_PAGE         0x10
+
+// Assume that valid page is accessible
 static bool IS_MMPTE_VALID(uint64_t mmpte)
 {
     return VMI_GET_BIT(mmpte, 0);
 }
 
-static bool IS_MMPTE_TRANSITION(uint64_t mmpte)
+// Assume that MM_GUARD_PAGE shoulbe reset to page been accessible
+static bool IS_MMPTE_ACCESSIBLE(uint64_t protection)
 {
-    return !IS_MMPTE_VALID(mmpte) && VMI_GET_BIT(mmpte, 11);
+    return (protection & 0x7) && !(protection & MM_GUARD_PAGE);
```

Comment:
Could we use some named constant instead of `0x7`?

Reply:
@icedevml done!